fdjangodog/middleware.py

```diff
@@ -1,15 +1,25 @@
 import time
 
-from datadog import statsd
+from datadog import DogStatsd
 from django.conf import settings
-from django.core.urlresolvers import resolve, Resolver404
+
+try:
+    from django.urls import resolve, Resolver404
+except ImportError:
+    from django.core.urlresolvers import resolve, Resolver404
 
 try:
     from django.utils.deprecation import MiddlewareMixin
     base_class = MiddlewareMixin
 except ImportError:
     base_class = object
 
+if hasattr(settings, 'FDJANGODOG_STATSD_HOST'):
+    statsd_host = settings.FDJANGODOG_STATSD_HOST
+else:
+    statsd_host = "localhost"
+
+statsd = DogStatsd(host=statsd_host)
```

Comment:
A *tiny* personal nitpick, I'd prefer to keep 2 lines before class definitions ref. [PEP8](https://www.python.org/dev/peps/pep-0008/#blank-lines)